### PR TITLE
Updates to the Public Site

### PR DESCRIPTION
**Pull request by biyeun**
NOTE: Public site is currently at a hidden URL. This is just to get the bulk of the PR reviewed and g2g.

@NoahCarnahan @czue 

You can view the latest on staging.commcarehq.org/public/home

### CODE REVIEW

File: corehq/apps/public/views.py [hunk 1]
```diff
@@ -1,12 +1,89 @@
...
+
+
+class SolutionsPublicView(TemplateView):
+    urlname = 'public_solutions'
+    template_name = 'public/solutions.html'
+
+    def get_context_data(self, **kwargs):
+        kwargs['is_solutions'] = True
+        return super(SolutionsPublicView, self).get_context_data(**kwargs)
+
```
Comment (dimagimon) [inline corehq/apps/public/views.py]: W391 blank line at end of file

**Comment by czue** [comment]
offlining some questions with others, don't merge but feel free to review.

@biyeun any suggestions on how review should happen? I really have no idea how i'd approach this or whether it's even necessary to review closely.

**Comment by czue** [comment]
@biyeun is this still relevant or is it replaced by https://github.com/dimagi/commcare-hq/pull/6498?

**Comment by biyeun** [comment]
it's replaced. closing.

**Pull request closed without merging**